chia-tools/src/bin/analyze-chain.rs

```diff
@@ -94,6 +96,19 @@ fn main() {
             }
         };
 
+        let ftb = match block.foliage_transaction_block {
```

Comment:
Im not a fan of abbreviations like this since it imo always just makes reading code less intuitive. Why not just name it `foliage_transaction_block`?

Reply:
long variable names adds noise